_commands/tools/ffmpeg.md

```diff
@@ -2,22 +2,70 @@
 ---
 ffmpeg
 ----
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+
+ffmpeg is a multimadia framework used to record, convert, and stream audio and video.
 
 <!-- minimal example -->
 ~~~ bash
```

Comment:
Just an overall comment for this bit: it's convention to include a dollar sign before the command to show example usage

i.e. `$ ffmpeg ...` instead of just `ffmpeg ...`